site/en/blog/autoplay/index.md

```diff
@@ -0,0 +1,290 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Autoplay policy in Chrome
+description: >
+  Learn best practices for good user experiences with the new autoplay policies in Chrome.
+subhead: >
+  Improved user experience, minimized incentives to install ad blockers, and reduced data consumption
+date: 2017-09-13
+updated: 2021-05-25
+authors:
+  - beaufortfrancois
+tags:
+  - autoplay
+  - media
+---
+
+{% Aside %}The Autoplay Policy launched in Chrome 66 for audio and video
+elements and is effectively blocking roughly half of unwanted media autoplays in
+Chrome. For the Web Audio API, the autoplay policy launched in Chrome 71. This
+affects web games, some WebRTC applications, and other web pages using audio
+features. More details can be found in the [Web Audio API](#webaudio) section
```

Comment:
The "[Web Audio API]" link does not work.

![image](https://user-images.githubusercontent.com/634478/119479954-7487f580-bd51-11eb-9495-ee95fade08ff.png)

Reply:
I think it's fixed.